wirecard/oxid-ee
# WDPD-47 Payment in Advance payment method

### This PR

* Adds Payment in Advance payment method
* Adds Unit & Selenium tests

### How to Test

* Open the admin, go to _Extensions → Modules_ and activate the module
* Go to _Shop Settings → Payment Methods_ and activate _Wirecard Payment in Advance_
* Open the shop and make a purchase using _Wirecard Payment in Advance_ as a payment method
* Payment information data should be shown on Thank You page.

### NOTES

* In order to simulate payment, one can send a deposit transaction via Postman to Oxid's notification URL. Transaction is then created, and it is marked "matched" or "unmatched" (if parent transaction id is missing).

### Jira Links

* https://jira.parkside.at/browse/WDPD-47
